composer.json

```diff
@@ -13,14 +13,14 @@
 		"lint": "find . -name \\*.php -not -path './vendor/*' -print0 | xargs -0 -n1 php -l",
 		"psalm": "psalm.phar --threads=1",
 		"psalm:update-baseline": "psalm.phar --threads=1 --update-baseline",
-		"psalm:clear": "psalm.phar --clear-cache && psalm --clear-global-cache",
+		"psalm:clear": "psalm.phar --clear-cache && psalm.phar --clear-global-cache",
 		"psalm:fix": "psalm.phar --alter --issues=InvalidReturnType,InvalidNullableReturnType,MissingParamType,InvalidFalsableReturnType",
 		"test:unit": "phpunit -c tests/phpunit.xml"
 	},
 	"require-dev": {
 		"nextcloud/coding-standard": "^1.0",
 		"psalm/phar": "^4.10",
-		"christophwurst/nextcloud": "dev-master@dev",
-		"phpunit/phpunit": "^8"
+		"phpunit/phpunit": "^8",
```

Comment:
Will be best to move the testing tools to separated vendor context to prevent conflict of package versions between the required by a test tool and the version required by this app.

To do this will be necessary use the package https://github.com/bamarni/composer-bin-plugin, have usage example of this package at other apps, you can search by `bamarni/composer-bin-plugin` at Nextcloud organization to search by examples.

Reply:
Ok thanks, will look into it. It was already suggested to me earlier today.

Reply:
But maybe that's also something for a follow-up PR, don't you think so?

Reply:
No problem.